lib/resuelvefc/team_management/level_manager.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Resuelvefc.TeamManagement.LevelManager do
+  @spec relate_team_levels_to_goals(list(map()), map()) :: list(map())
+  def relate_team_levels_to_goals(players, level_team_configuration) do
+    Enum.map(players, fn player ->
+      Map.put(
+        player,
+        :goles_minimos,
+        Map.fetch!(level_team_configuration.level_team_goals, player["nivel"])
```

Comment:
You should know how the pattern matching works, 'cause it makes your code more elegant.

Here an example, we can check it together later...

```suggestion
  def relate_team_levels_to_goals(players, %{level_team_goals: team_levels} = _config) do
    Enum.map(players, fn %{"nivel" => level} = player ->
      Map.put(
        player,
        :goles_minimos,
        Map.fetch!(team_levels, level)
```